USAGE.md

```diff
@@ -150,7 +150,9 @@ Example of assuming the Geostore roles from a LINZ AWS account:
 
 You can communicate with a Geostore instance using either the low-level application programming
 interface (API) or the high-level command-line interface (CLI). The following documentation explains
-how to work with both.
+how to work with both. For an example of working with Python, see the
+`should_successfully_run_dataset_version_creation_process_with_multiple_assets` function in
```

Comment:
This is quite a big function which looks like most of it is creating dummy files. Is there a way to publish a URL to just the relevant bits showing how to call the lambda functions with python? Or a GIST or something?

Reply:
I agree for the visibility but the good thing about pointing to this unit test is that if the function call change, the test will have to be updated. Might be more difficult to notice if the example is somewhere else?

Reply:
You're right that it's big, but it's not creating "dummy" files. The files it creates are the dataset. So the test basically shows the whole life cycle from creating a dataset in a staging bucket, calling the Geostore, verifying that the Geostore imported everything correctly, and then tearing down the contents of the staging bucket. Any separate code, as Paul points out, will eventually get out of date.

Reply:
Okidoke. If you can find what you need in there @paulfouquet . All good. 🚀